facade/src/main/java/com/redhat/lightblue/migrator/facade/ServiceFacade.java

```diff
@@ -357,4 +397,19 @@ public TimeoutConfiguration getTimeoutConfiguration() {
     public void setTimeoutConfiguration(TimeoutConfiguration timeoutConfiguration) {
         this.timeoutConfiguration = timeoutConfiguration;
     }
+
+    /**
+     * Shutdown thread pool executor for all facades.
+     *
+     */
+    public static void shutdown() {
```

Comment:
True. But you still need a method to call. ServiceFacade.shutdown() is that method.